src/workloads/execution/PingCommand.yml

```diff
@@ -0,0 +1,49 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/product-perf"
+Description: >
+  This is a simple test that runs ping command on MongoDB to meassure the latency of command dispatch.
+
+Clients:
+  Default:
+    QueryOptions:
+      maxPoolSize: 500
+
+MaxPhases: &MaxPhases 1
+ActorTemplates:
+- TemplateName: PingTemplate
+  Config:
+    Name: {^Parameter: {Name: "Name", Default: "TwoThreadPing"}}
+    Type: RunCommand
+    Threads: {^Parameter: {Name: "Threads", Default: 1}}
+    Phases:
+      OnlyActiveInPhases:
+        Active: {^Parameter: {Name: "Active", Default: [0]}}
+        NopInPhasesUpTo: *MaxPhases
+        PhaseConfig:
+          Duration: {^Parameter: {Name: "Duration", Default: "30 seconds"}}
```

Comment:
30 seconds is quite short, maybe consider making the duration a bit longer (60 seconds).